jaseci_studio/__e2e__/logs.spec.ts

```diff
@@ -7,12 +7,12 @@ test("logs page loads", async ({ page }) => {
 
 test("logs messages load", async ({ page }) => {
   await page.addInitScript(() => {
-    window.localStorage.setItem("serverUrl", "http://mysite.com");
+    window.localStorage.setItem("serverUrl", "http://localhost:8005");
```

Comment:
@Gim3l what do these URL do? Would localhost here work with tests and would it work with production?

Reply:
Those are mock urls used for e2e testing. Yes, the localhost version works with the tests on github, the other link doesn't.